src/main/java/org/folio/inventory/dataimport/handlers/actions/UpdateHoldingEventHandler.java

```diff
@@ -73,6 +76,8 @@ public class UpdateHoldingEventHandler implements EventHandler {
   private final Storage storage;
   private final MappingMetadataCache mappingMetadataCache;
   boolean isPayloadConstructed = false;
+  private final List<PartialError> resultedErrorHoldings = new ArrayList<>();
```

Comment:
Is it ok to have these defined as class fields? It seems to me this class is instantiated on start up and will be shared for different import processes. If it's true, they should be defined as local variables.

Reply:
@tatsiana-tarhonskaya Yes, this is ok. There is a recursive mechanism for the Optimistic Locking cases. (If there is "expiredHoldings"-list is NOT empty after processing). These lists "resultedSuccessHoldings" and "resultedErrorHoldings" will be filled after each iteration of retry for the OL. Then they will be cleared. So, finally, these fields should be as a class fields.

**BUT** this is really  good point from you about "different import processes.". I need to improve it that these lists will be not affected by different di-jobs. Thanks! Working on it!